assets/js/modules/analytics/components/dashboard/LegacyAnalyticsDashboardWidgetTopLevel.js

```diff
@@ -149,7 +148,7 @@ function LegacyAnalyticsDashboardWidgetTopLevel( { data, requestDataToState } )
 				<DataBlock
 					className="overview-total-users"
 					title={ __( 'Unique Visitors from Search', 'google-site-kit' ) }
-					datapoint={ readableLargeNumber( totalUsers ) }
+					datapoint={ !! totalUsers ? totalUsers : 0 }
```

Comment:
The value for the first split-second is `false` which is not valid against the new `PropTypes` definition for `DataBlock`. Since this is a legacy component, let's not bother improving this further, just disallowing `false` will fix that error.